Source/WebCore/css/parser/CSSParserContext.cpp

```diff
@@ -99,6 +99,7 @@ CSSParserContext::CSSParserContext(const Document& document, const URL& sheetBas
 #if ENABLE(CSS_PAINTING_API)
     , cssPaintingAPIEnabled { document.settings().cssPaintingAPIEnabled() }
 #endif
+    , cssTextUnderlinePositionLeftRightEnabled { document.settings().cssTextUnderlinePositionLeftRight() }
```

Comment:
Don't quite understand why the settings flag does not include enabled???

Reply:
Feel free to fix that here if you want, totally optional though.

Reply:
Probably something that was overlooked when naming the setting?